src/main/java/org/apache/aurora/scheduler/mesos/DockerContainerTasks.java

```diff
@@ -53,6 +55,10 @@ public static void configureTask(
 		ImmutableMap<String, String> envVariables = ImmutableMap.of(
 				"AURORA_TASK_ID", task.getTaskId(),
 				"AURORA_TASK_INSTANCE", Integer.toString(task.getInstanceId()),
+				"AURORA_JOB", Joiner.on('/').join(
+						serverInfo.getClusterName(),
+						JobKeys.canonicalString(taskConfig.getJob()),
+						task.getInstanceId()),
```

Comment:
Is there any issue with avoiding the explicit Integer.toString() conversion here?

Reply:
Not really, join() receives a list of Object and calls toString() on each. In the case of the instanceId it gets boxed into an Integer object.